go-controller/pkg/ovn/libovsdbops/chassis_private.go

```diff
@@ -0,0 +1,21 @@
+package libovsdbops
+
+import (
+	"context"
+	"fmt"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/sbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+)
+
+func FindChassisPrivate(sbClient libovsdbclient.Client) ([]sbdb.ChassisPrivate, error) {
```

Comment:
Please add a docblock for this! Also please add a note that if `chassisPrivate` objects are every used/manipulated by modelClient that it's indexes/ other relevant getUUID/setUUID functions will need to be added in `libovsdbops/model.go`

Reply:
Thanks for this insight. Added:
// FindChassisPrivate returns a list of ChassisPrivate from cache. Returned objects are read-only
// because getUUID and setUUID are not configured for ChassisPrivate table in model.go.